docs/code-search/code-navigation/auto_indexing.mdx

```diff
@@ -0,0 +1,320 @@
+# Auto-indexing
+
+<p className="subtitle">Learn and understand how auto-indexing works.</p>
+
+<Callout type="note">This feature is in beta for self-hosted customers.</Callout>
+
```

Comment:
@mmanela Technically, auto-indexing is enabled by default for Cloud customers (c.f. related discussion on how many executors to make available to customers). However, we're missing these bullets from our [GA definition](https://handbook.sourcegraph.com/departments/engineering/sdlc/).
- Performance optimized.
- Come with scenario-complete docs + samples.
- Sales and support staff trained and ready to go.

Should we change this to 'in beta for both self-hosted and Cloud customers' or just 'in beta'?